OTPKit/Services/LocationManagerService.swift

```diff
@@ -0,0 +1,191 @@
+//
+//  LocationManagerService.swift
+//  OTPKit
+//
+//  Created by Hilmy Veradin on 18/07/24.
+//
+
+import Foundation
+import MapKit
+import SwiftUI
+
+public final class LocationManagerService: NSObject, ObservableObject {
```

Comment:
I move all the location-services based to one LocationManagerService. What do you think of this approach? I'm kinda confused about this since I'm afraid it will be bloated one day, but if I separate this into many classes I'm afraid it will make an unnecessary abstraction? What do you think? @aaronbrethorst

Reply:
Let's not worry about over-bloating eventually. For now, I think this is a totally reasonable approach. https://understandlegacycode.com/blog/refactoring-rule-of-three/#premature-refactoring-is-the-root-of-many-bad-things

Reply:
Got it. Thanks @aaronbrethorst!